koala/cogs/base/api.py

```diff
@@ -14,9 +14,18 @@
 
 # Constants
 BASE_ENDPOINT = 'base'
-ACTIVITY_ENDPOINT = 'activity'
-SET_ACTIVITY_ENDPOINT = 'setActivity'
-SCHEDULE_ACTIVITY_ENDPOINT = 'scheduleActivity'
+ACTIVITY_ENDPOINT = 'scheduled-activity' # GET
+SET_ACTIVITY_ENDPOINT = 'activity' # PUT
+SCHEDULE_ACTIVITY_ENDPOINT = 'scheduled-activity' # POST
```

Comment:
Please update constant naming in line with the new value (e.g. ACTIVITY_ENDPOINT, SCHEDULED_ACTIVITY_ENDPOINT)

Also, since 2 of these endpoints are identical they can use the same constant